docs/development.md

```diff
@@ -90,12 +90,23 @@ Terminology we use:
 
 ### Reviewing a pull request
 
+#### The Pull Request review process
+
+1. The author submits a PR
+2. The @openshift-ci-robot automatically suggests an approver and requests 2 reviews from reviewers based on OWNERS files
+3. Reviewer goes and does the review.  
+    Once reviewer is ok with the changes he posts `/lgtm` as a comment to the PR. This will apply lgtm label.
+4. Approver goes and does the review.  
+    Once the review is done and the approver accepts the changes and pots `/approve` as a comment to the PR. This will apply approve label.
```

Comment:
![81vVSJ676yL _SY450_](https://user-images.githubusercontent.com/6422176/55187060-6e485f80-516e-11e9-81d5-afd10215c2aa.jpg)
 pots?

I think you mean posts :)

Reply:
https://giphy.com/gifs/drama-cat-EX6wKXVM9IOlO